src/Sulu/Bundle/MediaBundle/Resources/js/containers/MediaVersionUpload/MediaVersionUpload.js

```diff
@@ -0,0 +1,138 @@
+// @flow
+import React, {Fragment} from 'react';
+import {observer} from 'mobx-react';
+import type {IObservableValue} from 'mobx';
+import {action, observable, when} from 'mobx';
+import {Button, Loader} from 'sulu-admin-bundle/components';
+import {ResourceStore} from 'sulu-admin-bundle/stores';
+import {translate} from 'sulu-admin-bundle/utils';
+import MediaUploadStore from '../../stores/MediaUploadStore';
+import SingleMediaUpload from '../SingleMediaUpload';
+import CropOverlay from './CropOverlay';
+import FocusPointOverlay from './FocusPointOverlay';
+import mediaDetailsStyles from './mediaVersionUpload.scss';
+
+type Props = {|
+    resourceStore: ResourceStore,
+|};
+
+@observer
+class MediaVersionUpload extends React.Component<Props> {
+    mediaUploadStore: MediaUploadStore;
+    @observable showFocusPointOverlay: boolean = false;
+    @observable showCropOverlay: boolean = false;
+    showSuccess: IObservableValue<boolean> = observable.box(false);
```

Comment:
This is not doing anything anymore... We have to think about how to solve it, but the way it currently is, we could as well delete it...

Reply:
I think instead of this we should introduce an `onSuccess` callback, which can optionally be passed to the `Form` and is passed down to this component. @chirimoya Would you agree on the name?

Reply:
I have added the onSuccess callback as you suggested.